pkg/tar/tar.go

```diff
@@ -395,7 +395,7 @@ func (t *stiTar) ExtractTarStreamFromTarReader(dir string, tarReader Reader, log
 	})
 
 	if err != nil {
-		glog.Error("Error extracting tar stream")
+		glog.Error("Error extracting tar stream: %v", err)
```

Comment:
the %v doesn't appear to do what you intend (note: I modified the timeout to hit this case in my clone)?

```
ERROR: Error extracting tar stream: %vfunction timed out after 30s
```

Reply:
https://github.com/openshift/source-to-image/pull/932

Reply:
dangit lol https://github.com/openshift/source-to-image/pull/933